gulpfile.js

```diff
@@ -1,88 +1,15 @@
-var autoprefixer = require('gulp-autoprefixer');
-var closureCompiler = require('gulp-closure-compiler');
-var concat = require('gulp-concat');
 var gulp = require('gulp');
-var plumber = require('gulp-plumber');
-var runSequence = require('run-sequence');
-var sass = require('gulp-sass');
-var uglify = require('gulp-uglify');
+var wrench = require('wrench');
```

Comment:
I noticed on https://www.npmjs.com/package/wrench "Warning: This Project Is Deprecated!" -- should we use `fs-extra` instead?